### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by mcqueary**
Some elaboration on some NATS use cases, and fixing a couple typos.

@ColinSullivan1 @kozlovic  @wallyqs @aricart @bjflanne

### CODE REVIEW

**Comment by ShawnMilo** [comment]
Excellent. It made me think of "new" (to me) ideas for using NATS that wouldn't have occurred to me. Ideally this will do the same for others, and cause them to stop and play with it instead of closing the browser tab more frequently.

**Comment by mcqueary** [comment]
@ShawnMilo Thanks again for bringing it up -- our website and docs benefit so much from exactly that kind of input from the community!

**Comment by bjflanne** [comment]
Thanks @ShawnMilo for the diligence and time spent providing (actionable) feedback been great syncing up on this today!

File: content/documentation/index.md [hunk 1]
```diff
@@ -38,15 +38,15 @@ The core principles underlying NATS are performance, scalability, and ease-of-us
...
-- Load balancing
-- N-way scalability
-- Location transparency
-- Fault tolerance
+Some of the use cases and requirements that are ideal for NATS include:
+
+- **High throughput message fanout** - a small number of data producers (publishers) need to frequently send data to a much larger group of consumers (subscribers), many of whom share common interest in specific data sets or categories (subjects)
+- **Addressing, discovery** - sending data to specific application instances, devices, or users, or discovering all application instances/devices/users that are connected to your infrastructure. 
+- **Command and control (control plane)** - sending commands to running applications/devices and receiving status back from applications/devices, e.g. SCADA, satellite telemetry, IOT.
+- **Load balancing** - your application(s) produces a large volume of work items or requests and you would like to use a dynamically scalable pool of worker application instances to ensure you're meeting SLAs or other performance targets.
```
Comment (ColinSullivan1) [inline content/documentation/index.md]: One could also distribute components of a workflow for scalability and parallelization.

Comment (mcqueary) [inline content/documentation/index.md]: Are you suggesting that as a new bullet item, or elaboration on 'load balancing' ?

**Pull request merged**